src/utils/calls/nonces.ts

```diff
@@ -30,23 +30,6 @@ export const cancelOrderNonces = (
   };
 };
 
-export const cancelSubsetNonces = (
```

Comment:
Missed this last time, why was it removed?

Reply:
Because we don't support subset nonces in our api.